src/events/WindkraftForschung.ts

```diff
@@ -0,0 +1,33 @@
+import { defineEvent } from "../Factory"
+import { lawIsAccepted } from "../lawTools"
+import { markdown } from "../lib/utils"
+import { modify } from "../params"
+
+export default defineEvent({
+  title: "Durchbruch in der Windenergie-Forschung",
+  description: `Wissenschaftler haben eine neue Art der Energiegewinnung durch Wind entwickelt, mit der bestehende Windturbinen deutlich mehr Leistung bringen.`,
+
+  apply() {
+    return [modify("popularity").byValue(4), modify("electricityWind").byPercent(30)]
+  },
+
+  probability(game) {
+    const factor = lawIsAccepted(game, "WindkraftVereinfachen") ? 1.5 : 1
+    if (lawIsAccepted(game, "AusschreibungsverfahrenfuerWindkraftVerachtfachen")) {
+      return 0.5 * factor
+    }
+    if (lawIsAccepted(game, "AusschreibungsverfahrenfuerWindkraftVerdoppeln")) {
+      return 0.3 * factor
+    }
+    return 0
```

Comment:
What if "WindkraftVervierfachen" is currently the accepted one? (If i remember correctly, only one of them is active at the same time.

Reply:
Thanks, I simply forgot that.